### ISSUE TITLE
Questions about data

### ISSUE THREAD

**Issue by pat-alt** [gtfintechlab/fomc-hawkish-dovish#7]
Hi there, 

This is a very interesting dataset, thank you for sharing. I have a few questions:

1. In the filtered datasets, does the 'score' column correspond to the softmax output for the predicted label?
2. For the training datasets here in this repo, what is the difference between files named, for example, '...-split-combine-...' and simply '...-combine-...'?
3. Also for the training datasets, I assume the final digits in the filenames just indicate the seeds?

Thanks for your help!
